src/components/recruting/_02_prepare/_02/GroupIdeal.tsx

```
@@ -0,0 +1,175 @@
+import React, { useState, useRef } from "react";
+import { useForm } from "react-hook-form";
+
+export default function GroupIdeal() {
+  const [groupIdeals, setGroupIdeals] = useState<GroupIdeals>(() => {
+    const groups = ["기획", "개발", "디자인"];
+    return groups.reduce(
+      (acc, group) => ({
+        ...acc,
+        [group]: {
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)